src/resources/CrawlingModule/CrawlingModuleInterfaces.ts

```diff
@@ -57,6 +57,7 @@ export interface CreateCrawlingModuleLogRequestModel {
     instanceId: string;
     logType: CrawlingModuleLogRequestLogType;
     operationId: string;
+    id?: string;
```

Comment:
I couldn't find the call on swagger, should it really be optional?

Reply:
[Platform internal](https://platformdev.cloud.coveo.com/docs/internal?urls.primaryName=Connectivity#/Crawling%20Module/rest_organizations_paramId_crawlingmodules_paramId_logrequests_post)